askama_derive/src/generator.rs

```diff
@@ -1273,29 +1273,24 @@ impl<'a> Generator<'a> {
         if name == "json" {
             return Err("the `json` filter requires the `serde-json` feature to be enabled".into());
         }
-        #[cfg(not(feature = "serde-yaml"))]
-        if name == "yaml" {
-            return Err("the `yaml` filter requires the `serde-yaml` feature to be enabled".into());
-        }
 
-        const FILTERS: [&str; 2] = ["safe", "yaml"];
-        if FILTERS.contains(&name) {
+        let display_wrap = if name == "safe" {
             buf.write(&format!(
                 "{CRATE}::filters::{}({}, ",
                 name, self.input.escaper
             ));
+            DisplayWrap::Wrapped
```

Comment:
Please move this into a separate commit if you want it (Honestly not sure if I like it.)

Reply:
Hehe, seems like we have the opposite opinion on this matter.